### PR TITLE
feat: add `name` as projectName to cloned project template package.json

### PR DESCRIPTION
**Pull request by BRAVO68WEB**
- feat: add `name` as projectName to cloned project template package.json

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -146,6 +146,20 @@ async function main() {
       }`,
     )
   }
+  
+  const packageJsonPath = path.join(targetDirectoryPath, 'package.json')
+  const packageJson = fs.readFileSync(packageJsonPath, 'utf-8')
```
Comment (yusukebe) [inline src/index.ts:151]: Handle the error if a `pacakage.json` is not found. For example, the `deno` starter does not have `pacakge.json`.

**Review by yusukebe — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (BRAVO68WEB) [reply src/index.ts:151]: Sure man

**Comment by yusukebe** [comment]
@BRAVO68WEB 

Thank you for updating it. But still, I got the error when choosing the `deno`:

CleanShot 2024-06-13 at 11 21 11@2x

Can you see it?

**Comment by BRAVO68WEB** [comment]
Got it, my bad I used a different logic for checking for package json.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Pull request merged**